doc/source/release/1.11.0-notes.rst

```diff
@@ -131,108 +131,98 @@ New features
 
 `scipy.stats` improvements
 ==========================
-- Added `scipy.stats.sobol_indices`, a method to compute sensitivity indices of
-  Sobol.
-- Added `scipy.stats.dunnett`, a function that performs Dunnett's test of the
-  means of multiple experimental groups against the mean of a control group.
-- Added `scipy.stats.ecdf` for computing the empirical CDF and complementary
+
+New Features
+------------
+- `scipy.stats.sobol_indices`, a method to compute Sobol' sensitivity indices.
+- `scipy.stats.dunnett`, which performs Dunnett's test of the means of multiple
+  experimental groups against the mean of a control group.
+- `scipy.stats.ecdf` for computing the empirical CDF and complementary
   CDF (survival function / SF) from uncensored or right-censored data. This
   function is also useful for survival analysis / Kaplain-Meier estimation.
-- Added `scipy.stats.fdr_control` for adjusting p-values to control the false
-  discovery rate of multiple tests.
-- Added Filliben's test of whether a sample was not drawn from a hypothesized
-  distribution as ``method='Filliben'`` of `scipy.stats.goodness_of_fit`.
-- Added handling of censored data to the ``fit`` method of univariate
-  continuous distributions for fitting by maximum likelihood estimation. A
-  new class, ``CensoredData``, is added to represent censored data.
-- `scipy.stats.anderson_ksamp` now performs a permutation version of the test
-  if the user provides argument ``n_resamples``. The is particularly useful
-  when the p-value would be beyond the range of the tabulated values.
```

Comment:
I removed this because it has been reverted for an interface modification. gh-18470 would add it back.